src/model.js

```diff
@@ -51,35 +51,32 @@ const Model = function(_data) {
         return resultProxy;
     }
     // Watch changes via proxy
-    this.data = newProxy(_data, '');    
+    this.data = newProxy(_data, '');
 }
+
+const jsPathRegex = /([^[.\]])+/g;
+
 // get attribute
 Model.prototype.get = function(key, fallback = null)

```

Comment:
OT: Is there are reason your default `fallback` value is `null` rather than `undefined`? Standard getter procedure would use `undefined` to align with the native response from `({}).prop === undefined`

```suggestion
Model.prototype.get = function(key, fallback)
```

Typescript wouldn't be happy with the above as no default value is provided, so you could instead use:

```suggestion
Model.prototype.get = function(key, fallback = undefined)
```